### ISSUE TITLE
I-D Submission: confirmation of E-Mail sent has wrong TZ info

### ISSUE THREAD

**Issue by cabo** [ietf-tools/datatracker#5546]
### Describe the issue

I just submitted an I-D at 2023-04-27 05:58:36 UTC and got a confirmation

``` 
Email was sent at 2023-04-27 05:58:36 PST8PDT 
```

(That is in the future.)

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

---
Aggregated with agent assistance on 2026-08-18

**Comment by rjsparks** [comment]
could you please forward the entire email message you received to me?

**Comment by jennifer-richards** [comment]
Problem is here: https://github.com/jennifer-richards/datatracker/blob/b691cb4a9f85866a175890260b80be215461b372/ietf/utils/mail.py#L327

**Comment by cabo** [comment]
Indeed, the problem wasn't in the mail, it was in the Weg page that confirmed the submission and showed that the mails had been sent.
